recipe/build.sh

```diff
@@ -2,12 +2,11 @@
 
 # Ensure our geos will be used.
 rm -rf $SRC_DIR/geos-3.3.3
```

Comment:
Here there would be the same comment as with the bat script. The `geos-3.3.3` is removed from `basemap` because `basemap` does not provide the GEOS source code anymore. Therefore, there is nothing to remove.